### PR TITLE
MAINT: Update ``shims.material_import``

### PR DESCRIPTION
**Pull request by dipinknair**
Update with deprecation warning

### CODE REVIEW

File: src/ansys/mechanical/core/embedding/shims.py [hunk 1]
```diff
@@ -1,40 +0,0 @@
...
-but workarounds exist in an older release
-"""
-
-
-def import_materials(
-    app: "ansys.mechanical.core.embedding.Application", material_file: str
-) -> None:
-    """Import material from matml file."""
-    if app._version >= 232:
-        materials = app.DataModel.Project.Model.Materials
```
Comment (koubaa) [inline src/ansys/mechanical/core/embedding/shims.py:35]: keep the shims - users might be using them.  just add an obsolete warning to it.

**Review by koubaa — CHANGES_REQUESTED** [review]

File: src/ansys/mechanical/core/embedding/shims.py (hunk 1 above)
Comment (koubaa) [reply src/ansys/mechanical/core/embedding/shims.py:35]: This isn't correct, the jscript workaround is what we have to remove!

Comment (dipinknair) [reply src/ansys/mechanical/core/embedding/shims.py:35]: I have updated and added tests as well.

File: tests/embedding/test_app.py [hunk 2]
```diff
@@ -292,12 +292,19 @@ def test_building_gallery(pytestconfig, run_subprocess, rootdir):
 
 
 @pytest.mark.embedding
-def test_rm_lockfile(embedded_app, tmp_path: pytest.TempPathFactory):
```
Comment (koubaa) [inline tests/embedding/test_app.py:295]: why did you remove test_rm_lockfile?

Comment (dipinknair) [reply tests/embedding/test_app.py:295]: It was not intended, I think I was adding this above it

**Review by koubaa — APPROVED** [review]
